genesis/src/main.rs

```diff
@@ -97,12 +97,14 @@ fn main() -> Result<(), Box<dyn error::Error>> {
         default_lamports_per_byte_year,
         default_rent_exemption_threshold,
         default_rent_burn_percentage,
+        default_base_rent_per_year,
     ) = {
         let rent = Rent::default();
         (
             &rent.lamports_per_byte_year.to_string(),
             &rent.exemption_threshold.to_string(),
             &rent.burn_percent.to_string(),
+            &rent.base_rent_per_year.to_string(),
```

Comment:
this isn't necessary, just pick some number of bytes that an account is assumed to use